comid/entity_test.go

```diff
@@ -50,24 +50,24 @@ func TestEntity_Valid_name_regid_and_roles(t *testing.T) {
 }
 
 func TestEntities_Valid_empty(t *testing.T) {
-	e := Entity{}
-	tv := NewEntities().AddEntity(e)
+	e := &Entity{}
```

Comment:
Since the Extensions are now built in the collection, i.e. Entities as you say, should we also not add the Unit Tests to exercise the examples for extensions in `entity_test.go` Far easier for readers to understand at one place, 
- How Native Entity is tested
- How Extension in the Entity is tested ?

Reply:
`Entities` is a thin wrapper around `Collection`; there is nothing to test -- it just calls through to `Collection` for everything. Collection is tested by `collection_test.go`.

For example of extensions we have the `example_profile_test.go`.

Reply:
May be I was not clear, let me try to explain it again:
What I meant was multiple extensions via Entities, i.e. `Instantiate` Two Entity Structures via Collections, each have a different extension inside it, and set, get and assert the extensions are exercised correctly, inside Entities Collection. This way, readers of Entity know: how to exercise Entity specific Extensions.

Also then there is also a case for not using Extensions but still how to set Multiple Entities, - with no extensions. 

Perhaps this should have been covered outside of the scope of this PR, but might be helpful to add it at some point in time?

Reply:
This scenario is already covered by `exampole_profile_test.go` (with mesaurements, rather than entities, but the same principle applies).